src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -0,0 +1,81 @@
+"""The Oracle dialect.
+
+This inherits from the ansi dialect.
+"""
+from sqlfluff.core.dialects import load_raw_dialect
+from sqlfluff.core.parser import (
+    BaseSegment,
+    Ref,
+    Sequence,
+    StartsWith,
+    OneOf,
+)
+
+ansi_dialect = load_raw_dialect("ansi")
+oracle_dialect = ansi_dialect.copy_as("oracle")
+
+oracle_dialect.sets("unreserved_keywords").difference_update(["COMMENT"])
+oracle_dialect.sets("reserved_keywords").update(["COMMENT", "ON"])
+
+
+# Adding Oracle specific statements.
+@oracle_dialect.segment(replace=True)
+class StatementSegment(BaseSegment):
+    """A generic segment, to any of its child subsegments."""
+
+    type = "statement"
+
+    parse_grammar = ansi_dialect.get_segment("StatementSegment").parse_grammar.copy(
+        insert=[
+            Ref("CommentStatementSegment"),
+        ],
+    )
+
+    match_grammar = ansi_dialect.get_segment("StatementSegment").match_grammar.copy()
+
+
+@oracle_dialect.segment()
+class CommentStatementSegment(BaseSegment):
+    """A `Comment` statement.
+
+    COMMENT [text]
+    https://docs.oracle.com/cd/B19306_01/server.102/b14200/statements_4009.htm
+    """
+
+    type = "comment_statement"
+
+    match_grammar = StartsWith(Sequence("COMMENT", "ON"))
+
+    parse_grammar = Sequence(
+        "COMMENT",
+        "ON",
+        Sequence(
+            OneOf(
+                Sequence(
+                    OneOf(
+                        "TABLE",
+                        "VIEW",
+                    ),
+                    Ref("TableReferenceSegment"),
+                ),
```

Comment:
Looking at the docs it looks like `COMMENT ON VIEW ...` isn't a thing?
Both views and tables get use `COMMENT ON TABLE ...`.
Can you check this?
![image](https://user-images.githubusercontent.com/80432516/149345828-4f49f9ad-4f1d-490e-9c3f-2829d09f5ae1.png)

Reply:
Confirmed it is not a thing, thank you 🙏